features/journalist_can_create_article.feature

```diff
@@ -0,0 +1,33 @@
+Feature: Journalist can create articles
+
+    As a journalist
+    In order write articles
+    I would like to be able to create an article
+
+    Background: 
+        Given the following users exist
+            | email          | password | role       |
+            | jocke@craft.se | password | journalist |
+            | alecia@craft.se| password | visitor    |
+        And I am logged in as "jocke@craft.se"
+
+    Scenario: Journalist can create article
+        When I visit the journalist page
+        And I fill in "Title" with "Vikings living amongst us"
+        And I fill in "Lead" with "Hurra"
+        And I fill in "Content" with "Vad kul"
+        And I click "Save Article"
+        Then I should see "Article was successfully created."
+
+    Scenario: Journalist can't create article [Sad path]
+        When I visit the journalist page
+        And I fill in "Title" with "Vikings living amongst us"
+        And I fill in "Lead" with "Hurra"
+        And I click "Save Article"      
+        # Then I should not see "Article was successfully created."
```

Comment:
We need some assistance on this step, it still "shows" on the page even though we cant see it

Reply:
Please use this step.